apps/web/pages/_document.jsx

```diff
@@ -8,6 +8,11 @@ export default function Document(props) {
     <Html className="h-full scroll-smooth bg-gray-100 font-normal antialiased" lang="en">
       <Head>
         <meta name="color-scheme"></meta>
+        <link rel="preconnect" href="https://fonts.googleapis.com" />
```

Comment:
- The fonts should be loaded from the included files, please remove external resource reference
- If this fixes sth. please create a new issue

Reply:
For the fonts, I wanted to set the font with of the completed date to medium weight but the current font available doesn't have medium weight.

It's either 'bold' or 'regular'

Reply:
Thats by design to keep it clean for now